refactor: fixing offchain/onchain naming

benesjan opened the pull request:
We were inconsistent in our naming:
For the meaning of `offchain` we used the following:
- off-chain
- offchain
- out-of-band

And for `onchain` we used
- on-chain
- onchain

Decided to just use `offchain` and `onchain` consistently. I think out-of-band term was genuinely harmful when it comes to external devs understanding what's going on.

benesjan commented:
* **#16946** <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/16946?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#16942** <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/16942?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/16942?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `next`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

benesjan commented on l1-contracts/src/core/Rollup.sol:
```diff
@@ -546,15 +546,15 @@ contract Rollup is IStaking, IValidatorSelection, IRollup, RollupCore {
    * @notice  Get the proposer for the slot at a specific timestamp
    *
-   * @dev     This function is very useful for off-chain usage, as it easily allow a client to
+   * @dev     This function is very useful for offchain usage, as it easily allow a client to
```
@LHerskind are you fine with me merging these changes to the contracts given that they were audited? Can revert this if you prefer.

benesjan replied:
Got a DM that Lasse is fine with this.

nventuro reviewed (commented):
Given there's so many instances of these, maybe consider adding cspell rules to prevent regressions?

benesjan commented:
> Given there's so many instances of these, maybe consider adding cspell rules to prevent regressions?

@nventuro added in https://github.com/AztecProtocol/aztec-packages/pull/16942/commits/debd73b16fa42618dc280eb3a337363bacfa6304

benesjan committed:
refactor: fixing offchain/onchain naming

We were inconsistent in our naming:
For the meaning of `offchain` we used the following:
- off-chain
- offchain
- out-of-band

And for `onchain` we used
- on-chain
- onchain

Decided to just use `offchain` and `onchain` consistently. I think out-of-band term was genuinely harmful when it comes to external devs understanding what's going on.
